src/js/module/main/main.js

```diff
@@ -3,6 +3,8 @@ import MainViewDialog from 'module/main/dialog';
 import Const from 'const';
 import Button from '../component/button';
 
+const IS_PLANTING_CLASS = 'is-planting';
```

Comment:
This class follows [github naming convention](http://primercss.io/guidelines/#class-naming-conventions). Thats true, it should be prefixed with `plantingjs`.